internal/app/api/repositories/user/user_repo.go

```diff
@@ -310,6 +310,23 @@ func (r *MongoRepository) CountUserUsersAvailable(ctx context.Context, ignoreIds
 	return r.collection().CountDocuments(ctx, query)
 }
 
+// this method help get update verify email in mail collection
+func (r *MongoRepository) UpdateMailVerified(ctx context.Context, email string) error {
```

Comment:
Can we move this to mail repo?

Reply:
Mail Verified was updated after the user successful signup, so I think it should be in user repo...
If we move this to mail repo, maybe create 1 API  update, I think it not good